test/unit/specs/components/ActionModal/utils/actions.js

```diff
@@ -8,6 +8,23 @@ export const sendTx = {
       }
     ]
   },
+  txDelProps: {
+    toAddress: "cosmos123",
+    amounts: [
+      {
+        amount: "20000",
+        denom: "uatom"
+      }
+    ],
+    validator_address: "cosmosvaloper1pnmn05d9prestn2vxujr8h88fpm9c0skeg72p5",
+    message: {
+      value: {
+        validatorMoniker: "operator_account_2"
```

Comment:
This data structure looks weird. "Message.value.validatorMoniker" is that intended?

Reply:
Yes, it was intended due to the way the action managers message was structured. But It does not matter because the way that it is structured now this test will be removed as it is the same functionality for any message.